examples/data-sources/indykite_application_agent/data-source.tf

```diff
@@ -1,6 +1,33 @@
+# Example - all parameters shown (still valid)
 data "indykite_application_agent" "application_agent" {
-  app_space_id    = "ApplicationSpaceGID"
-  app_agent_id    = "ApplicationAgentGID"
-  name            = "AppAgentName"
-  api_permissions = ["Authorization", "Capture"]
+  app_space_id = "ApplicationSpaceGID"
+  app_agent_id = "ApplicationAgentGID"
+  name         = "AppAgentName"
 }
```

Comment:
I think this is not valid. You can read by `app_agent_id` or by `app_space_id + name`. But not both together.

This is actually validated via `oneOfAppID := []string{nameKey, appAgentIDKey}`